src/gravity/params.yaml

```diff
@@ -1,28 +1,142 @@
 ---
 gravity:
   _description: "External gravitational sources"
-  type:
-    _type: string
-    _description: "Type of gravitational source"
-    uniform:
-      _type: opt
-      _description: "Uniform gravitational acceleration"
-    point:
-      _type: opt
-      _description: "Point mass source."
-    binary:
-      _type: opt
-      _description: "Binary mass source."
-    nbody:
-      _type: opt
-      _description: "N point mass sources. Requires an <nbody> node"
   gm:
     _type: Real
     _description: "Normalization parameter setting GM."
+    _default: "0.0"
```

Comment:
Default is `Big` (or maybe changed to `NaN`) right?

Reply:
I suppose it should be nan